pkgs/development/libraries/libavif/default.nix

```diff
@@ -35,13 +39,21 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
+    gdk-pixbuf
     libaom
     zlib
     libpng
     libjpeg
     dav1d
   ];
 
+  postInstall = ''
+    mkdir -p ${placeholder "gdkPixbufLoader"}/share/thumbnailers
+    cat ${./thumbnailer} | sed "s|@bindir@|${gdk-pixbuf}/bin|g" > ${placeholder "gdkPixbufLoader"}/share/thumbnailers/libavif.thumbnailer
+  '';
+
+  outputs = [ "out" "gdkPixbufLoader" ];
```

Comment:
This would typically go after pname + version, see see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.